src/lib/components/ui/ArticleList.svelte

```
@@ -27,6 +28,7 @@
     let visibleArticles = $state(articlesPerPage);
     let previousVisibleCount = $state(articlesPerPage);
     let loading = $state(false);
+    let newArticleRef: HTMLElement | null = null;
```

ESLint

[error] 'newArticleRef' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)